src/main.rs

```diff
@@ -11,9 +11,111 @@ use std::marker::PhantomData;
 use french_numbers::*;
 
     /// These constants are to avoid magic strings/values.
-    const LANGUAGE_LOCALES: &[&str; 18] = &["en", "es", "bg", "de", "eo", "fr", "gr", "hi", "ie", "jp", "la", "nl", "pl", "pt", "ru", "sk", "tr", "zh"];
+    const LANGUAGE_LOCALES: &[&str; 19] = &["en", "es", "bg", "bw", "de", "eo", "fr", "gr", "hi", "ie", "jp", "la", "nl", "pl", "pt", "ru", "sk", "tr", "zh"];
     const LANGUAGES_DIRECTORY: &str = "translations";
     const MSG: &str = "msg";
+    
+    /// These constants are necessary numerical values. Our extensive list has 100 integers, so far.
+    const one: i64 = 1;
+    const two: i64 = 2;
+    const three: i64 = 3;
+    const four: i64 = 4;
+    const five: i64 = 5;
+    const six: i64 = 6;
+    const seven: i64 = 7;
+    const eight: i64 = 8;
+    const nine: i64 = 9;
+    const ten: i64 = 10;
+    const eleven: i64 = 11;
+    const twelve: i64 = 12;
+    const thirteen: i64 = 13;
```

Comment:
```suggestion
```

The number 13 is unlucky and may violate memory safety

Reply:
I did not even realize. I will change that.

Reply:
```diff
/*delete unsafe number*/
-const thirteen: i64 = 13; 
+const thirteen: i64 = 12+1;
```

Reply:
> ```diff
> /*delete unsafe number*/
> -const thirteen: i64 = 13; 
> +const thirteen: i64 = 12+1;
> ```

Perhaps this should use hex encoding to avoid the number from being even stored in memory?

Reply:
It's a difficult question. The number 13 is unpredictable. And recording it in a different form can become even more dangerous. We do not need the number 13, but the value of the expression 12 + 1. It's not 13 and it's safe. You can write it in hexadecimal form, but it will be magical, magic is not allowed! Therefore, we should add a named kolnstandu that denotes the sum of numbers that represent the neutralization of a dangerous number in the form of representing its value as the sum of safe ones, and only then we need to set this new constant to the current one. I think we should write a generator that will automatically generate a code for providing a safe option for the value of 13, which will substitute its result into the constant we need.

Could rust be so? If not, then this is definitely a fundamental bug in the language!

Reply:
This is a wonderful idea, I suggest trying it and if not, opening a RFC

Reply:
Let's try.